vignettes/dust.Rmd

```diff
@@ -47,6 +47,7 @@ There are two main parts here; the class (`walk`) and an interface function (`du
 The class is what does most of the work. It is important that this does not use anything in the R api as it may be called in parallel. Therefore use only C++ standard library types.  It has to provide every public type or method as the example above:
 
 * the type `real_t` has to exist and indicate what sort of numeric type you use for "real numbers". Typically this will be `double`,but if you need a different type (longer or shorter) you can use it here.
+* the type `data_t` which for now should match exactly the construction `typedef dust::no_data data_t` until we stabilise and document this interface
```

Comment:
Is there anywhere in the vignette where `has_openmp` needs changing? Or did this remain unchanged as there was still an R function which digs into capabilities and checks this, with the same function name as before

Reply:
yeah, the public interface is unchanged